code/game/gamemodes/miniantags/revenant/revenant_blight.dm

```diff
@@ -56,12 +53,8 @@
 				affected_mob << "<span class='revenbignotice'>You feel like [pick("nothing's worth it anymore", "nobody ever needed your help", "nothing you did mattered", "everything you tried to do was worthless")].</span>"
 				affected_mob.adjustStaminaLoss(45)
 				PoolOrNew(/obj/effect/overlay/temp/revenant, affected_mob.loc)
-				if(affected_mob.dna && affected_mob.dna.species)
-					affected_mob.dna.species.handle_mutant_bodyparts(affected_mob,"#1d2953")
-					affected_mob.dna.species.handle_hair(affected_mob,"#1d2953")
```

Comment:
I don't see the point of changing our hair color when we're giving the mob a color value that will color everything.